### ISSUE TITLE
Define Goals

### ISSUE THREAD

**Issue by danielgroves** [danielgroves/danielgroves.net#12]
Content is king, and all that. Before starting to work out how to lay things out properly, or how it should look, I really need to figure out exactly what it is I want to achieve with this effort. 

An initial off-the-top-of-my-head brainstorm:

- To make people more aware of my work as a photographer
- To make people more aware of who I am and what I do
- To help place my images in front of more people
- To help showcase my images, what goes into them and and what they can teach you about a place. 
- To help insure further people, through optimised blog-posts and pushing them more

**Comment by nickcharlton** [comment]
Something I've struggled with before when reviewing posts is finding where it is. What do you think about the existing set of "high level" categories?

**Comment by danielgroves** [comment]
Yep, this is something I've been considering. I had loosely formed the idea of having 'Notebook', and then 4-5 main categories below that (adventure, photography, fear, etc) and allowing filtering within each by tag. 

I'm also wondering about extending the 'series' functionality further, but possibly more in the long run once I've got everything into a dynamic environment to give myself some more flexibility (thinking long-term Ansel switch here). 

I've also been wondering about breaking the 'development' section out into a different site entirely so my main site can become more focused. I reserved danielgroves.github.io some time ago with this thought. What's your opinion on that?

**Comment by nickcharlton** [comment]
Categories sounds like a good approach to this. It'll keep everything together, but clearly delineate things.

I'd run with the series thing. It works well for a lot of what you write. It ends up similar to Digital Ocean's Series (e.g.: https://www.digitalocean.com/community/tutorial_series/getting-started-with-freebsd).

I'd avoid splitting the site. You'll end up with two projects to maintain, not own that domain yourself and split traffic between your interests.

**Comment by danielgroves** [comment]
Cool, thanks @nickcharlton 👍 

Incidentally, added the start of a series thing over the summer already: https://danielgroves.net/notebook/2016/04/eurovelo#series-information
